ansible/ocs.yaml

```diff
@@ -0,0 +1,292 @@
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-0 --source {{ ocs_device }}2 --target vdc --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-0 --source {{ ocs_device }}3 --target vdd --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-1 --source {{ ocs_device }}5 --target vdb --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-1 --source {{ ocs_device }}6 --target vdc --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-1 --source {{ ocs_device }}7 --target vdd --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-2 --source {{ ocs_device }}8 --target vdb --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-2 --source {{ ocs_device }}9 --target vdc --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-2 --source {{ ocs_device }}10 --target vdd --persistent"    
+
+    - name: Fixes for dev-scripts vbmc container
+      block:
+      - name: Stop and remove vbmc container
+        shell: podman stop vbmc && podman rm vbmc
+
+      - name: Start vbmc container with proper mounts
+        shell: podman run -d --net host --privileged --name vbmc -v "{{ base_path }}/virtualbmc/vbmc":/root/.vbmc -v "/root/.ssh":/root/ssh -v "/var/run/libvirt:/var/run/libvirt:Z" quay.io/metal3-io/vbmc 
+
+    - name: Create and start storage VM vbmcs
+      shell: |
+        vbmc add "{{ ocp_cluster_name}}-storage-{{ item }}" --address 192.168.111.1 --port "626{{ item }}" --username ADMIN --password ADMIN;
+        vbmc start "{{ ocp_cluster_name}}-storage-{{ item }}"
+      with_items:
+        - 0
+        - 1
+        - 2
+
+    - name: Open ports for storage VM vbmcs in firewalld
+      firewalld:
+        port: "626{{ item }}/udp"
+        permanent: yes
+        state: enabled
+        zone: libvirt
+        immediate: yes
+      with_items:
+        - 0
+        - 1
+        - 2
+    
+    - name: Shutdown storage VMs using IPMI via vbmcs
+      shell: ipmitool -I lanplus -U ADMIN -P ADMIN -H 192.168.111.1 -p "626{{ item }}" power off
+      retries: 2
+      with_items:
+        - 0
+        - 1
+        - 2
+
+    - name: Prepare DNS/DHCP for storage VMs for dev-scripts environment
+      shell: |
+        virsh net-update {{ ocp_cluster_name }}bm add ip-dhcp-host "<host mac='52:54:00:82:69:6{{ item }}' name='storage-{{ item }}' ip='192.168.111.3{{ item }}' />" --live --config;
+        virsh net-update {{ ocp_cluster_name }}bm add dns-host "<host ip='192.168.111.3{{ item }}'><hostname>storage-{{ item }}</hostname></host>" --live --config
+      register: add_dhcp_dns_entries
+      failed_when: add_dhcp_dns_entries.stderr != "" and "there is an existing" not in add_dhcp_dns_entries.stderr
+      with_items:
+        - 0
+        - 1
+        - 2
+
+    - name: Get existing {{ ocp_cluster_name }} worker machineset JSON
```

Comment:
It looks like from here downwards you need to start a new play for localhost with the oc_local role.